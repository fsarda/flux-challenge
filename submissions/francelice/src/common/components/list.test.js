import React from 'react';
import List from './list';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

describe('actions', () => {

  test(`MyComponent renders with default props`, () => {
    const wrapper = shallow(<List />);
    expect(wrapper).toMatchSnapshot();
  });

  test('List has childs if content is not empty', () => {
    const component = renderer.create(
      <List content={[1,2,3]}></List>,
    );
    let tree = component.toJSON();
    expect(tree.children).not.toBeNull();
  });
  

});
