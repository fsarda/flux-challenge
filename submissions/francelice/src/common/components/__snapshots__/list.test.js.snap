// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions MyComponent renders with default props 1`] = `<ul />`;
